cypress/locators/checkbox/index.js

```diff
@@ -1,13 +1,13 @@
+import {
+  REVERSE_CHECKBOX, FIELD_HELP_INLINE_CHECKBOX, CHECKBOX_HELP,
+  CHECKBOX_COMMOM_INPUT_FIELD, CHECKBOX_LABEL,
+} from './locators';
+
 // knobs locators
-const REVERSE_CHECKBOX = '#reverse';
-const FIELD_HELP_INLINE_CHECKBOX = '#fieldHelpInline';
 export const reverseCheckbox = () => cy.get(REVERSE_CHECKBOX);
 export const fieldHelpInlineCheckbox = () => cy.get(FIELD_HELP_INLINE_CHECKBOX);
 
 // component preview locators
-const CHECKBOX_HELP = '.carbon-checkbox__help-text';
-const CHECKBOX_COMMOM_INPUT_FIELD = '.carbon-checkbox > .common-input__field';
-const CHECKBOX_LABEL = '.common-input__label';
 export const checkboxHelpTextPreview = () => cy.iFrame(CHECKBOX_HELP);
 export const checkboxCommonInputField = () => cy.iFrame(CHECKBOX_COMMOM_INPUT_FIELD);
```

Comment:
typo in selector CHECKBOX_COMMOM_INPUT_FIELD

Reply:
fixed